README.md

```diff
@@ -1,22 +1,47 @@
 # ansible-mps-plugin
+The ansible-mps-plugin is one of many plugins of the [DeMAF](https://github.com/UST-DeMAF) project.
+It is designed to transform [Ansible deployment models](https://docs.ansible.com) into an [EDMM](https://github.com/UST-EDMM) representation.
 
-## Build
-```shell
-./mvnw package
-```
+The plugin only works (without adaptions) in the context of the entire DeMAF application using the [deployment-config](https://github.com/UST-DeMAF/deployment-config).
+The documentation for setting up the entire DeMAF application locally is [here](https://github.com/UST-DeMAF/EnPro-Documentation).
+
+## Build and Run Application
+
+You can run the application without the [deployment-config](https://github.com/UST-DeMAF/deployment-config) but it will not run as it needs to register itself at the [analysis-manager](https://github.com/UST-DeMAF/analysis-manager).
+
+If you want to boot it locally nevertheless use the following commands.
 
-## Run
 ```shell
 ./mvnw spring-boot:run
 ```
-or (requires build):
+or:
```

Comment:
Maybe add "[...] to use the built package"?

Reply:
done